frontend/uv-poc/src/app/home/home.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="bg-dark p-5">
+  <div class="container">
+    <span class="display-4 text-light ml-8 text-center"><p>Remember</p></span>
+    <span class="text-muted text-center">
+      <h2>
+        <p>this is just a proof of concept, don't save production data</p>
```

Comment:
`don't save production data`  isn't a valid statement for this application as it only reads data, doesn't write anything.

Reply:
This is just a sample data. We can discuss this once we discuss how the wireframes should look like.